rust/src/modularity.rs

```diff
@@ -109,12 +109,10 @@ fn modules_impl(key: &str, opts: &InstallOpts) -> Result<()> {
         -1,
         gio::NONE_CANCELLABLE,
     )?;
-    let reply_child = crate::variant_utils::variant_tuple_get(reply, 0)
+    let reply = reply
+        .get::<(String,)>()
```

Comment:
This syntax looks somehow bizarre to me, is it equivalent to `(String, _)` or does it mean something else?

Reply:
Yeah it is funny at first¹ - it's a 1-tuple whose element is a String.   One doesn't often see 1-tuples.  Basically in DBus wire protocol return values are *always* tuples - the single element case is `(s)` not `s`.

¹ But after you see it once, it makes sense

Reply:
Difference between 1-tuple and the type itself

Reply:
Ooooh, somehow I never encountered/realized there is this syntax ambiguity between defining 1-ary tuples and putting a type within plain parenthesis, TIL. Is it called out in the language reference: https://doc.rust-lang.org/reference/types/tuple.html.